parlai/mturk/webapp/dev/app.jsx

```diff
@@ -34,6 +35,93 @@ function resolveState(state_string) {
   return state;
 }
 
+function postData(url = ``, data = {}) {
+  // Default options are marked with *
+  return fetch(url, {
+      method: "POST", // *GET, POST, PUT, DELETE, etc.
+      mode: "cors", // no-cors, cors, *same-origin
+      cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+      credentials: "same-origin", // include, *same-origin, omit
+      headers: {
+          "Content-Type": "application/json; charset=utf-8",
+          // "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
nit: delete comment?

Reply:
Ooops this was copypasta from the fetch api docs